package.json

```diff
@@ -42,6 +42,7 @@
   "dependencies": {
     "@babel/runtime": "^7.6.0",
     "appium-android-driver": "^4.20.0",
+    "appium-awtk-driver": "^0.x",
```

Comment:
Please remove the driver from dependencies, but keep the other stuff. The idea is users could add the necessary package(s) themselves if they need it. See, for example, the discussion on https://github.com/appium/appium/pull/15227
We are currently working on Appium 2, which could load drivers dynamically, while Appium 1 would be deprecated anyway.

Reply:
Got it. I've deleted it. 
Looking forward to Appium 2！